### PR TITLE
Add support for new denominator format

### LINKED ISSUE #188: Support inverse denominator
Support inverse denominator for migrated USDC pool: https://github.com/zkBob/zkbob-contracts/pull/79/files#diff-e323fb0df158445703db898a290a14791771e48f31c6f10add5bb86df87311f8R152

### PR DESCRIPTION
**Pull request by lok52**
Closes #188

### CODE REVIEW

File: zp-relayer/utils/helpers.ts [hunk 1]
```diff
@@ -248,3 +248,9 @@ export function getFileHash(path: string) {
   hash.update(buffer)
   return hash.digest('hex')
 }
+
+export function applyDenominator(n: BN, d: BN) {
+  return d.isNeg()
+    ? n.div(d.neg())
+    : n.mul(d)
+}
```
Comment (k1rill-fedoseev) [inline zp-relayer/utils/helpers.ts:256]: Maybe this way is shorter?
```suggestion
export function applyDenominator(n: BN, d: BN) {
  return d.testn(255)
    ? n.div(d.maskn(255))
    : n.mul(d)
}
```

Comment (lok52) [reply zp-relayer/utils/helpers.ts:256]: But we also need to do a reverse of `applyDenominator` here, in `FeeManager`, which is easier to do with simple `.neg()` call
https://github.com/zkBob/zeropool-relayer/blob/2ae48d9e7e4a539ce59b3f56a4cb42b5989d696e/zp-relayer/services/fee/FeeManager.ts#L48

Comment (lok52) [reply zp-relayer/utils/helpers.ts:256]: I mean, that BN.js doesn't store the sign in 255th bit, as far as I know

Comment (lok52) [reply zp-relayer/utils/helpers.ts:256]: But, I guess we still can do a xor here, in `FeeManager`, instead of `.neg()`

Comment (lok52) [reply zp-relayer/utils/helpers.ts]: I refactored it a bit, as you suggested

**Review by k1rill-fedoseev — APPROVED** [review]

**Review by EvgenKor — APPROVED** [review]
lgtm

**Review by akolotov — APPROVED** [review]

**Pull request merged**